### ISSUE TITLE
Difference between getTtaIncludingTraffic and getTtaExcludingTraffic is too low

### ISSUE THREAD

**Issue by lvanyal** [heremaps/here-android-sdk-examples#304]
I created route from Palo Alto to Oregon Zoo with  
`dynamicPenalty.trafficPenaltyMode = Route.TrafficPenaltyMode.OPTIMAL
`
After 3 routes calculated I got these results:

```
route.getTtaIncludingTraffic(WHOLE_ROUTE).duration = 63463, 66244, 70823
route.getTtaExcludingTraffic(WHOLE_ROUTE).duration = 63461, 66242, 70821
```

Is there any additional parameter I missed for better traffic calculation?

**Comment by starand** [comment]
Hi @lvanyal 
This is an issue which should be fix in scope of SDK 3.15 release.

**Comment by starand** [comment]
Fix is included in SDK 3.15.
